types/meteor/email.d.ts

```diff
@@ -1,20 +1,10 @@
+import { SendMailOptions } from 'nodemailer';
+
 declare module 'meteor/email' {
     namespace Email {
-      interface EmailOptions {
-        from?: string | undefined;
-        to?: string | string[] | undefined;
-        cc?: string | string[] | undefined;
-        bcc?: string | string[] | undefined;
-        replyTo?: string | string[] | undefined;
-        subject?: string | undefined;
-        text?: string | undefined;
-        html?: string | undefined;
-        headers?: Object | undefined;
-        attachments?: Object[] | undefined;
-        mailComposer?: MailComposer | undefined;
-      }
+      type EmailOptions = { mailComposer: MailComposer } | SendMailOptions
```

Comment:
`interface`s have one potential advantage - they can be extended in the application (thanks to declaration merging). Using `SendMailOptions` here is good idea, but it should remain an `interface`.

Reply:
Is it actually useful to be able to extend `EmailOptions`? `SendMailOptions` _is_ the type that Meteor accepts (it just passes it on to nodemailer), and this accurately captures things like the fact that `mailComposer` is mutually exclusive with any other options.

Reply:
I don't know 🤷 If you suggest there's no need for that, I'm fine with that too.

Reply:
I guess there are two cases where it might be useful to add additional options - if you have a custom transport or if you register hook functions which mutate the email. I can add an additional interface that can be overridden and gets mixed into `EmailOptions`